presto-yarn-test/bin/run_on_docker.sh

```diff
@@ -33,8 +33,9 @@ function check_hive() {
 }
 
 function run_product_tests() {
+  MVN_VERSION=$(mvn -q -Dexec.executable="echo" -Dexec.args='${project.version}' --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
```

Comment:
In `presto-product-tests` I did it differently. See https://github.com/prestodb/presto/blob/master/presto-product-tests/bin/run_on_docker.sh#L146
and https://github.com/prestodb/presto/blob/master/presto-product-tests/src/main/resources/presto.env
and https://github.com/prestodb/presto/blob/master/presto-product-tests/pom.xml#L108